src/constants/docsComponentMapping.js

```diff
@@ -62,7 +62,17 @@ export const components = {
       }
     }
   `,
-  h3: styled(TextComponents.H3)`
+  h3: styled(TextComponents.LinkedH3)`
+    a {
+      color: ${({ theme }) => theme.text};
+      @media (${MEDIA_QUERIES.canHover}) {
+        &:hover {
+          color: ${({ theme }) => theme.darkGrey};
+        }
+      }
+    }
+  `,
+  h4: styled(TextComponents.LinkedH4)`
```

Comment:
I thought we'd [deliberately stopped linking h3/h4s](https://github.com/stellar/new-docs/pull/92/files#diff-52089f61bc4808567aee5c5950816cdb)? Change looks good just want to make sure it's behavior we want

Reply:
I noticed on [glossary - fee bumps](https://developers.stellar.org/docs/glossary/fee-bumps/) that [Replace by Fee](https://developers.stellar.org/docs/glossary/fee-bumps/#replace-by-fee) which is H4 is being linked to. There were links for `<H3/>` as well.

Reply:
Okay sweet! That makes sense